vantage6.sh

```diff
@@ -95,6 +95,18 @@ if ! id -nG $LOGNAME | grep -qw docker; then
     usermod -aG docker $LOGNAME 2>&1 | tee -a $REPORT
 fi
 
+## Add a root crontab to reset permissions on the vantage6 docker volumes after
+## reboot. This is necessary because docker resets the permissions on startup
+## and we are then not able to make new Redcap data available to a running
+## Docker container.
+echo '(7/7) ### Add crontab to reset vantage6 docker folder permissions on reboot'
+current_crontab="/tmp/my_current.crontab"
+crontab -l > $current_crontab
+# sleep 30 seconds to prevent racing condition with Docker startup
+echo "@reboot sleep 30 && bash $SCRIPT_DIR/set_docker_folder_permissions.sh" >> $current_crontab
```

Comment:
I had another thought. Wouldn't it be better to check if the docker deamon is running (=initialized)? Maybe one of the following suggestions in the `set_docker_permissions.sh` could remove the `sleep 30` command?